### PR TITLE
Allow setting HOST that server binds to (e.g., for Tor-only mode or for testing)

### PR DESCRIPTION
**Pull request by xanoni**
Added a new environment variable named "HOST" (default: localhost) and updated README.md

This builds on top of https://github.com/BlueWallet/LndHub/pull/280

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,7 +47,10 @@ app.use('/static', express.static('static'));
 app.use(require('./controllers/api'));
 app.use(require('./controllers/website'));
 
-let server = app.listen(process.env.PORT || 3000, function () {
-  logger.log('BOOTING UP', 'Listening on port ' + (process.env.PORT || 3000));
+const bindHost = (process.env.HOST || 'localhost')
```
Comment (Overtorment) [inline index.js]: default should be `0.0.0.0`

File: index.js [hunk 2]
```diff
@@ -47,7 +47,10 @@ app.use('/static', express.static('static'));
...
+const bindPort = (process.env.PORT || 3000)
```
Comment (Overtorment) [inline index.js]: semicolons missing. eslint wont be happy

File: README.md [hunk 3]
```diff
@@ -11,10 +11,22 @@ You can use those guides or follow instructions below:
...
+ln -si "${LND_DATA_DIR}/tls.cert" ./
+ln -si "${LNDHUB_CFG}" ./config.js
+
+rm -rf ./build || true
+mkdir ./build
+
+babel ./ --out-dir ./build --copy-files --ignore node_modules
+
+export HOST='lndhub.mydomain.com' && export PORT=3000
+node build/index.js
```
Comment (Overtorment) [inline README.md]: pls revert all changes to readme

File: index.js (hunk 2 above)
Comment (xanoni) [reply index.js]: sorry, was as vi (not even vim, because that didn't exist) job on some remote box ;-)

File: index.js (hunk 1 above)
Comment (xanoni) [reply index.js]: K. Wanted to be conservative (that's why I explained in the README how to set the port), but I was at least 60% sure I would get this feedback (given I'm changing defaults).

File: README.md (hunk 3 above)
Comment (xanoni) [reply README.md]: Please see my other comments in the PR that you closed. I think the correct way to handle the README would be:

1. Fix the instructions to mention the `npm start` step
2. Add a disclaimer saying that some of the instructions from the linked articles may be outdated (explicitly calling `babel`)
3. Mention the `HOST` and `PORT` environment variables

I'll put that together unless you disagree. Let me know please.

File: index.js [hunk 4]
```diff
@@ -47,7 +47,10 @@ app.use('/static', express.static('static'));
 app.use(require('./controllers/api'));
 app.use(require('./controllers/website'));
 
-let server = app.listen(process.env.PORT || 3000, function () {
-  logger.log('BOOTING UP', 'Listening on port ' + (process.env.PORT || 3000));
+const bindHost = process.env.HOST || '0.0.0.0';
+const bindPort = process.env.PORT || 3000;
+
+let server = app.listen(bindPort, bindHost, function () {
+  logger.log('BOOTING UP', 'Listening on ' + bindHost + ':' + bindPort);
```
Comment (Overtorment) [inline index.js:54]: lgtm

File: README.md [hunk 5]
```diff
@@ -1,64 +1,100 @@
...
 
 
-
 ## Responsible disclosure
 
-Found critical bugs/vulnerabilities? Please email them to bluewallet@bluewallet.io
-Thanks!
+Found critical bugs/vulnerabilities? Please email them to bluewallet@bluewallet.io.
+
+Thank you!
```
Comment (Overtorment) [inline README.md]: pls revert readme changes completely

Comment (xanoni) [reply README.md]: you really like your current README ;-)

Comment (xanoni) [reply README.md]: @Overtorment commit dropped ...

**Review by Overtorment — APPROVED** [review]

**Pull request merged**